### PR TITLE
Update docs with new styles, layout

### PR DESCRIPTION
**Pull request by jquense**
Copies the basic theme from reactjs.org and the structure/layout from getbootstrap.com

### CODE REVIEW

**Comment by jquense** [comment]
obviously not done, we still need to update components to v4 and whatnot to fix styles here but I think this is fine for now.

File: src/Container.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ class Container extends React.Component {
         {...props}
         className={classNames(
           className,
-          bsPrefix,
+          !fluid && bsPrefix,
```
Comment (taion) [inline src/Container.js]: maybe better to write as:


```js
fluid ? `${bsPrefix}-fluid` : bsPrefix
```

or even

```js
`${bsPrefix}${fluid ? '-fluid' : ''}`
```

File: www/gatsby-config.js [hunk 2]
```diff
@@ -73,6 +73,9 @@ module.exports = {
     },
     'gatsby-plugin-catch-links',
     'gatsby-plugin-sass',
-    'gatsby-plugin-less',
+    {
+      resolve: 'gatsby-plugin-css-literal-loader',
```
Comment (taion) [inline www/gatsby-config.js]: ooo

File: www/src/css/global.scss [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, Courier New, monospace;
+  background-color: transparentize($note, 0.75);
+  color: $text;
+  padding: 0.1rem;
+}
+
+// .bs-docs-nav {
```
Comment (taion) [inline www/src/css/global.scss]: remove?

File: www/src/css/theme.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+$text: #1a1a1a; // very dark grey / black substitute
+$subtle: #6d6d6d; // light grey for text
+$subtleOnDark: #999;
+$divider: #ececec; // very light grey
+$note: #ffe564; // yellow
+$error: #ff6464; // yellow
+$white: #ffffff;
+$black: #000000;
+
+// $lightShade: #dfd9d3;
```
Comment (taion) [inline www/src/css/theme.scss]: ?

File: www/src/layouts/index.js [hunk 5]
```diff
@@ -20,7 +14,7 @@ function DefaultLayout({ children, location }) {
 
       {children}
 
-      <PageFooter />
+      {/* <PageFooter /> */}
```
Comment (taion) [inline www/src/layouts/index.js]: ?

File: www/src/pages/components/modal.js [hunk 6]
```diff
@@ -1,22 +1,30 @@
...
 import ModalBasic from '../../examples/Modal/Basic';
-import ModalContained from '../../examples/Modal/Contained';
 import ModalDefaultSizing from '../../examples/Modal/DefaultSizing';
 import ModalCustomSizing from '../../examples/Modal/CustomSizing';
 import ModalVerticallyCentered from '../../examples/Modal/VerticallyCentered';
 import ModalGrid from '../../examples/Modal/Grid';
 import withLayout from '../../withLayout';
 
+const styles = css`
+  /* has to be fully global becausse of modal portals */
```
Comment (taion) [inline www/src/pages/components/modal.js]: "because"

File: www/src/pages/getting-started/introduction.js [hunk 7]
```diff
@@ -13,216 +14,203 @@ export default withLayout(
...
-define(['react-bootstrap'], function(ReactBootstrap) {
-var Alert = ReactBootstrap.Alert;
-...
-});`}
-              </code>
-            </pre>
-            <h3>
-              <Anchor id="browser-globals">Browser globals</Anchor>
-            </h3>
+                <!-- Optional theme -->
```
Comment (taion) [inline www/src/pages/getting-started/introduction.js]: remove this section... no longer relevant for TWBS4

File: www/package.json [hunk 8]
```diff
@@ -31,11 +32,14 @@
     "react-dom": "^16.4.1",
     "react-helmet": "^5.2.0",
     "react-live": "^1.9.2",
+    "react-tackle-box": "^1.2.0",
```
Comment (taion) [inline www/package.json]: unused?

File: www/package.json [hunk 9]
```diff
@@ -31,11 +32,14 @@
...
+    "recompose": "^0.28.0",
     "shakespeare-data": "^2.5.0",
     "yup": "^0.24.1"
   },
   "devDependencies": {
     "@babel/preset-flow": "^7.0.0-beta.40",
+    "@fortawesome/fontawesome-free": "^5.2.0",
```
Comment (taion) [inline www/package.json]: unused? looks like we use cdn

File: www/package.json [hunk 10]
```diff
@@ -10,11 +10,12 @@
     "bootstrap": "^4.0.0",
     "classnames": "^2.2.5",
     "common-tags": "^1.8.0",
+    "css-literal-loader": "^0.5.2",
     "formik": "^0.11.11",
     "gatsby": "^2.0.0-beta.59",
     "gatsby-link": "^2.0.0-beta.5",
     "gatsby-plugin-catch-links": "^2.0.2-beta.2",
-    "gatsby-plugin-less": "^2.0.0-beta.5",
+    "gatsby-plugin-css-literal-loader": "^1.0.0",
```
Comment (taion) [inline www/package.json]: the split between production and dev dependencies here is odd :p

**Pull request merged**